### DISCUSSION TITLE
Playbook entries in SOC alerts show incorrect timestamp

### DISCUSSION THREAD

**Discussion by greatapoc** [Security-Onion-Solutions/securityonion#3589] [Unsupported Versions] [answered]
2.3.40 Standalone install (sneaky upgrade)

Every entry in SOC Alerts coming from Playbook shows 0001-01-01 10:00:00 as the timestamp and no other information unless you drilldown. Also none of the other fields like source/destination IP are populating. Looks like Playbook drilldowns should reference "event_data.*".

Also the kibana_pivot shows "https://securityonion/kibana/app/kibana#/discover?_g=()&_a=(columns:!(_source),interval:auto,query:(language:lucene,query:'_id:{[_id]}'),sort:!('@timestamp',desc))--" which comes back with no results matching "_id:{[_id]}"

![timestamps](https://user-images.githubusercontent.com/8947638/112076969-8d4c2180-8bcf-11eb-941e-60bfcd09973c.PNG)

**Comment by jot49** [comment, marked as answer]
Already reported several times
https://github.com/Security-Onion-Solutions/securityonion/discussions/2069

**Reply by defensivedepth** [reply]
Will be fixed in our next release :)
